omod/src/main/resources/liquibase.xml

```diff
@@ -196,6 +196,33 @@
 							 columnNames="name" />
 	</changeSet>
 	
+	<changeSet id="metadatamapping-2016-01-06-0800" author="jasonvena">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="metadatamapping_metadata_term_mapping"
+						  columnName="metadata_class" />
+		</preConditions>
+		<comment>
+			Make MetadataTerm.metadataClass optional. We follow the openmrs convention of checking if the column exists.
```

Comment:
You commented earlier: "Here we refer to the table name (rather than the class name) in the hibernate mapping"

I'm not sure what you mean by "table name" as the table name is "metadatamapping_metadata_term_mapping" and not "MetadataTerm". Referring to the actual table name is a good idea.

Btw, the misleading filename MetadataTerm.hbm.xml is actually my fault. I'll fix that later. Thanks for reminding me!